CHANGELOG.md

```diff
@@ -99,6 +102,11 @@ Additions to existing modules
                     updateAt (padRight m≤n x xs) (inject≤ i m≤n) f ≡ padRight m≤n x (updateAt xs i f)
   ```
 
+* In `Relation.Binary.Definitions`
+  ```agda
+  Directed _≤_ = ∀ x y → ∃[ z ] x ≤ z × y ≤ z
```

Comment:
Why binary? Don't you want to say that for any I-indexed family of points, there's a 'z' that is below all of them?